docker.go

```diff
@@ -89,6 +90,28 @@ func getIPsToNames(client dockerClienter, id string) (ipsToNamesMap, error) {
 			names = appendNames(names, name)
 		}
 
+		if label, ok := containerFull.Config.Labels["com.costela.docker-etchosts.add_hosts"]; ok {
```

Comment:
(see above)
```suggestion
		if label, ok := containerFull.Config.Labels["net.costela.docker-etchosts.extra_hosts"]; ok {
```
but maybe we could put this into a constant? At least the prefix?